docs/internal-documentation/maintenance/ReleaseProcess-create-test-distribution.md

```diff
@@ -69,7 +75,7 @@ MANIFEST.SKIP, i.e. no missing or extra files:
     make distcheck
 
 
-## 4. Produce distribution tarballs
+## 5. Produce distribution tarballs
```

Comment:
There are two `5` sections (this looks odd) :
`## 5. Produce distribution tarballs`
`## 5. Produce distribution zip file`

Maybe the second one should be a `5.1` :
 `### 5.1. Produce distribution zip file`

Reply:
It should then be 5 and 6 respectively. I will correct.